### PR TITLE
instantiate co-views out of this.render

### PR DESCRIPTION
**Pull request by coderhaoxin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,11 +36,13 @@ module.exports = function (path, opts) {
   // default extension to `html`
   if (!opts.default) opts.default = 'html';
 
-  if (opts) debug(fmt('options: %s', opts));
+  debug(fmt('options: %s', opts));
```
Comment (coderhaoxin) [inline index.js]: always pass

File: index.js [hunk 2]
```diff
@@ -72,7 +74,6 @@ module.exports = function (path, opts) {
       if (ext == 'html' && !opts.map) {
         yield send(this, join(path, file));
       } else {
-        var render = cons(path, opts);
```
Comment (coderhaoxin) [inline index.js]: should not instantiate `co-views` every `render`

**Comment by coderhaoxin** [comment]
ping @queckezz

**Comment by queckezz** [comment]
lgtm, thanks!

**Pull request merged**